modules/cli/pages/cbstats-intro.adoc

```diff
@@ -16,21 +16,14 @@ cbstats host:port [common options] command [command options]
 
 == Description
 
-The [.cmd]`cbstats` tool queries differently depending on the port:
+The [.cmd]`cbstats` tool will give you operations per node per bucket:
+
 
-* When using the Couchbase data port 11210, it will give you operations per node per bucket:
```

Comment:
it looks like Moxi port is 11211, just checking if this line needs to be removed here?